utils/build-presets.ini

```diff
@@ -1024,6 +1024,23 @@ skip-build-swift
 skip-build-cmark
 llvm-include-tests
 
+
+[preset: buildbot_incremental_linux,tsan-libdispatch-test]
+mixin-preset=buildbot_incremental_linux,llvm-only
+build-subdir=buildbot_incremental_tsan_libdispatch_test
+
+llvm-cmake-options=-DLLVM_ENABLE_PROJECTS=clang;libcxx;compiler-rt  \
+  -DCOMPILER_RT_INTERCEPT_LIBDISPATCH=ON  \
+  -DCOMPILER_RT_LIBDISPATCH_INSTALL_PATH=/home/yln/work/swift-source/build/buildbot_incremental_tsan_libdispatch_test/toolchain-linux-x86_64/usr
```

Comment:
`/home/yln/work/swift-source/build/buildbot_incremental_tsan_libdispatch_test/toolchain-linux-x86_64/usr` this should be provided by the build-script option

Reply:
For example: 
```
installable-package=%(installable_package)s
```

Reply:
But then it says that "var_name" not specified, if I don't. Note that I don't want to customize this path, it should just point to whichever directory my toolchain will reside.

Reply:
This dir will not be available on CI, `/home/yln/work/swift-source/build/buildbot_incremental_tsan_libdispatch_test/`